src/pybind/mgr/dashboard_v2/frontend/src/app/core/not-found/not-found.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="row">
+  <div class="col-md-12 text-center">
+    <h1>Sorry, we could not find what you were looking for</h1>
+    <h2>Instead, enjoy some cephalopods!!!</h2>
```

Comment:
I like the idea, but aren't we always displaying the same picture? So "some cephalopods" may be an exaggeration. Maybe over the top, but instead of showing a single image (and having to include it in the code base), how about embedding a Flickr slide show, e.g. the results of 

https://www.flickr.com/search/?text=cephalopod&license=2%2C3%2C4%2C5%2C6%2C9&media=photos&orientation=landscape

Reply:
I don't think that an admin of a production system wants to have an UI that accesses resources from outside.